lnclient/ldk/ldk.go

```diff
@@ -999,9 +999,10 @@ func (ls *LDKService) GetOnchainBalance(ctx context.Context) (*lnclient.OnchainB
 }
 
 func (ls *LDKService) RedeemOnchainFunds(ctx context.Context, toAddress string) (string, error) {
-	txId, err := ls.node.OnchainPayment().SendAllToAddress(toAddress)
+	spendableBalance := ls.node.ListBalances().SpendableOnchainBalanceSats
+	txId, err := ls.node.OnchainPayment().SendToAddress(toAddress, spendableBalance)
```

Comment:
The argument should be in millisats - can you multiply the spendable balance by 1000?

Reply:
I am wondering where this is going wrong because I tested this feature while developing it and the amount seems to be correct.

However, after multiplying the spendableBalance with 1000 to get the amount in msat, I get an error "not enough funds available". See this example with the current code I have:

1. Request 1M sats from mutinynet
tb1qnga845us4t4hafkp8aneq5f7tf9za9sr9u55f5
https://mutinynet.com/tx/7fc3a8cf78a3872774dcd6c112609915388ed586ba4ae7bc339446c78234487d
2. Savings balance updates to 999,786 sats
3. Generate new onchain address
tb1quq39g744c9e926tt6uzdrerq225d8pue9wffnh
3. Withdraw Savings Balance to this address
https://mutinynet.com/tx/a89dfa542b6efdd24d3c95738dc5a66dbc087a2917e19e51739164ba8e47ac6e
4. Spending balance updates to 999,572 sats
5. Send funds back to Mutinynet
tb1qd28npep0s8frcm3y7dxqajkcy2m40eysplyr9v
6. Savings balance updates to 0 sats

### Docs
https://sourcegraph.com/github.com/lightningdevkit/ldk-node/-/blob/src/payment/onchain.rs?L47
https://docs.rs/ldk-node/latest/ldk_node/payment/struct.OnchainPayment.html#method.send_to_address

Reply:
I created a bug in LDK, thanks for testing this!